tests/fixtures/rules/require-const-for-ember-properties/bad/destructuring.js

```diff
@@ -0,0 +1,5 @@
+// Let
+let { isNone } = Ember;
+
+// Var
+var { isNone } = Ember;
```

Comment:
Could you actually split up these two statements into separate files (one for `let` and one for `var`)? That way, we can be sure that both fail.